openshift-kubecost-install.md

```diff
@@ -503,13 +510,20 @@ cortextool rules print \
 Install Kubecost on your K8s cluster with Grafana Cloud Prometheus query endpoint and `dbsecret` you created in Step 4
 
 ```Bash
-
-helm upgrade -i -n kubecost kubecost ./cost-analyzer \
-    --set kubecostModel.promClusterIDLabel=cluster \
-    --set global.prometheus.fqdn=https://<REPLACE-WITH-GRAFANA-PROM-QUERY-ENDPOINT> \
-    --set global.prometheus.enabled=false \
-    --set global.prometheus.queryServiceBasicAuthSecretName=dbsecret
-
+export CLUSTER_ID="CLUSTER_OCP"
```

Comment:
I would include the same `Note` you used above about setting the `CLUSTER_OCP` values to something.

Reply:
Good catch and good idea. Thank you Sean, I fixed it in the latest commit before merging.